packer/create-support-bundle.sh

```diff
@@ -31,15 +31,44 @@ done
 
 mkdir -p "$LOG_DIR"
 
+k8s_cm_hostname_to_url() {
+  kubectl get configmap/studio --namespace $NAMESPACE -o=jsonpath="{.data.$1}" | sed -E 's#^(https?://)?([^:/?]+).*#\2#'
```

Comment:
Do we have `jq` installed? if not can we add it. it's more powerful than jsonpath or go-template filtering (go-template has very limited builtin functions available).
Then, you can do this, for example, to extract all keys with URL in it in one command:

```shell
kubectl -n studio get cm/studio -o=jsonpath="{.data}" | jq 'with_entries(if (.key|test("URL")) then ( {key: .key, value: .value | @base64d } ) else empty end )'
```

Reply:
Unfortunately, no `jq` installed. 

The example you shared is really nice. The only challenge is that we still need to convert those URLs to hostnames. Does `jq` support some kind of regex substitution?